daemon/src/db/cull_old_dlcs.rs

```diff
@@ -180,7 +179,16 @@ mod tests {
             EventKind::RolloverCompleted { dlc: None, .. },
         ));
 
-        assert_eq!(events[2].event, most_recent_rollover);
+        assert!(matches!(
+            &events[2].event,
+            EventKind::RolloverCompleted { dlc: Some(_), .. },
+        ));
+
+        let event0_timestamp = &events[0].timestamp;
+        let event1_timestamp = &events[1].timestamp;
+        let event2_timestamp = &events[2].timestamp;
+
+        assert!(event0_timestamp < event1_timestamp && event1_timestamp < event2_timestamp);
```

Comment:
🙃 I wouldn't test this here. This is testing a property of `load_cfd_events`: that the events returned are ordered. I would be happy to introduce a test in `db.rs` for this (in a separate commit if possible).